docs/source/advanced/no-runtime.mdx

```diff
@@ -60,28 +63,16 @@ To parse a network response into the type safe models, use `parseJsonResponse`:
 
 ```kotlin
 /**
- * responseSource should contain a json with data and possibly errors:
+ * jsonResponse should contain a json with data and possibly errors:
  * {
  *  "data": ...
  *  "errors": ...
  *  "extensions": ...
  * }
  **/
-val response = operation.parseJsonResponse(responseSource)
+val jsonReader = Buffer().write(jsonResponse).jsonReader()
+val response = operation.parseJsonResponse(jsonReader)
 
 println(response.data)
 ```
 
-## Converting a Query.Data model to/from JSON
-
-Symmetrically to the `composeJsonResponse` and `parseJsonResponse`, there is `composeJsonData` and `parseJsonData` to convert to/from data without errors or extensions:
-
-
-```kotlin
-var data = SomeQuery.Data(...)
-val jsonString = operation.composeJsonData(data);
-
-// parse back to models, it will match the initial data
-data = operation.parseJsonData(jsonString)
-```
```

Comment:
I removed this because I don't think anyone asked that question but I'm happy to bring it back if we think it's important. The only place I can think where this could be useful is for generating models from mocked json but then it most likely belongs somewhere else